src/core/public/chrome/ui/header/collapsible_nav.tsx

```diff
@@ -71,6 +71,45 @@ function getOrderedCategories(
   );
 }
 
+function getMergedNavLinks(
+  unknowns: ChromeNavLink[],
+  orderedCategories: string[],
+  categoryDictionary: ReturnType<typeof getAllCategories>
+): Array<string | ChromeNavLink> {
+  unknowns = sortBy(unknowns, 'order');
```

Comment:
It is not a good practice to modify the original parameters inside a function call, can we create a new temp variable to store the sorted unknowns links?